docs/synapse/userguides/storm_ref_lift.ipynb

```diff
@@ -623,6 +624,170 @@
     "_ = await core.fini()"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "hideCode": true,
+    "hideOutput": true
+   },
+   "outputs": [],
+   "source": [
+    "# Create a cortex for the Safe Lifts section\n",
+    "core = await getTempCoreCmdr()"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "hideCode": false
+   },
+   "source": [
+    "Safe Lifts\n",
+    "----------\n",
+    "\n",
+    "\"Safe\" lifts, also known as \"try\" lifts, are lifts performed in a safe and error-free manner. Safe lifts refer to lifts that perform :ref:`data-type` validation before attempting to lift a query node, and fails safely if an invalid Type is encountered. Specifically, it prevents a Cortex from throwing a runtime execution error, and terminating query execution when Type validation is unsuccessful for a given property value.\n",
+    "\n",
+    "When lifting nodes by property value using the equals (``=``) comparator, Type validation is not performed on the supplied property value prior to query execution. If a query is executed with an invalid Type, or a Type that does not correspond to the supplied form, the Cortex will throw a ``BadTypeValu`` error, and terminate the query as shown below."
```

Comment:
still confusing here - we always do that validation, we just allow those errors to tear down the runtime